engine/src/main/java/org/terasology/config/Config.java

```diff
@@ -67,6 +68,7 @@
  * Terasology user config. Holds the various global configuration information that the user can modify. It can be saved
  * and loaded in a JSON format.
  */
+@API
```

Comment:
This gives modules access to the path where the game is. That doesn't seem like that much of a security problem, but I'm not sure.

Reply:
Yep, this gives modules access to this class.
Rendering Modules needs it for read RenderingConfigs.